### ISSUE TITLE
0.84 Lovelace lost translations

### ISSUE THREAD

**Issue by sgttrs** [home-assistant/home-assistant-polymer#2354]
**Home Assistant release with the issue:** 0.84.3

**Last working Home Assistant release (if known):** 0.83

**Browser and Operating System:** Chrome on Windows 10 and Android 6.0

**Description of problem:** The first load of the Lovelace UI will always show only state, translates are not load. Translated states shows if state was changed.

**Additional information:**
Page loaded:
![1](https://user-images.githubusercontent.com/33804747/50142341-14fda400-02d4-11e9-9ef9-22cd971ad79e.png)
State changed:
![2](https://user-images.githubusercontent.com/33804747/50142362-25158380-02d4-11e9-9b33-c5961c4b47ab.png)

Related issues: https://github.com/home-assistant/home-assistant-polymer/issues/2308 https://github.com/home-assistant/home-assistant-polymer/issues/2292

**Comment by sgttrs** [comment]
https://github.com/home-assistant/home-assistant-polymer/issues/2308

**Comment by Anonym-tsk** [comment]
Related: https://github.com/home-assistant/home-assistant-polymer/pull/2293

**Comment by bieniu** [comment]
I have the same issue on 0.84.3 (Chrome on Windows and Opera on Linux). It occurs randomly. The browser cache was cleaned many times.

![received_366972777211558](https://user-images.githubusercontent.com/478555/50144934-a4915b80-02b0-11e9-9e62-0dd860567b42.png)

**Comment by sgttrs** [comment]
I just updated Hassio from 0.83.3 to 0.84.3 and it seems that this problem does not exist here. At the same time, the problem remains on Home Assistant installed via python venv.

**Comment by balloob** [comment]
I have been able to figure it out. The reason is actually quite interesting. The bug has been around for a long time, but we only noticed it now because we're rendering faster.

Internally, the frontend is caching the name and state display of states. It used to be that we didn't start rendering until the resources were loaded, because it was slow. Now that we're faster, we end up rendering before the language resources are loaded. This results in us rendering fallback states ("off") and then storing that in the cache.
